spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -1409,6 +1409,20 @@ def sso_token_for_video
       @sso_token_for_video ||= ref_data.sso_token_for_video
     end
 
+    # generates group hash used in the analytics time_series API
+    def group_for_view_by(view_by, measures)
+      if view_by.nil? || measures.nil?
+        return nil
+      else
+        return {
```

Comment:
Can we add a comment here something like `# Using rank=DESCENDING and limit=3 as this is what the App Store Connect analytics dashboard uses`?